bundles/org.openhab.binding.millheat/README.md

```diff
@@ -0,0 +1,109 @@
+# Millheat Binding
+
+This binding integrates the Mill Wi-Fi enabled panel heaters. 
+See https://www.millheat.com/mill-wifi/
+
+## Supported Things
+
+This binding supports all Wi-Fi enabled heaters as well as the Wi-Fi socket.
+
+* `account` = Mill Heating API - the account bridge
+* `heater` = Panel/standalone heater
+* `room` = A room defined in the mobile app
+
+## Discovery
+
+The binding will discover both rooms and heaters. 
+
+In order to do discovery, add a thing of type Mill Heating API and add username and password.
+
+## Thing Configuration
+
+See full example below for how to configure using thing files.
+
+### Account
+
+* `username` = email address used in app
+* `password` = password used in app
+* `refreshInterval` = number of seconds between refresh calls to the server 
+
+### Room
+
+* `roomId` = id of room. Use auto discovery to find this value
+
+### Heater
+
+* `macAddress` = network mac address of device. 
+Can be found in the app by viewing devices. 
+Or you can find it during discovery. 
+Used for heaters connected to a room
+* `heaterId` = id of device/heater. 
+Use auto discovery to find this value. 
+Used to identify independent heaters or heaters connected to a room
+* `power` = number of watts this heater is consuming when active. 
+Used to provide data for the currentPower channel.
+
+## Channels
+
+### Room channels
+
+| Channel             | Read/write    | Item type | Description |
+| ------------------- | ------------- | --------- | ----------- |
+| currentTemperature  | R             | Number    | Measured temperature in your room (if more than one heater then it is the average of all heaters) |
```

Comment:
Add the quantity unit to the item type.
```suggestion
| currentTemperature  | R             | Number:Temperature    | Measured temperature in your room (if more than one heater then it is the average of all heaters) |
```
This comment applies for all quantity items in this list.

Reply:
Done